pkg/k8s/client/clientset/versioned/clientset.go

```diff
@@ -12,12 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Code generated by client-gen. DO NOT EDIT.
```

Comment:
Why did this message disappeared? You used the `1.10.0-beta.3` generator, right?

Reply:
probably not, I think I had a 1.9 variant checked out (why that message would change I have no idea)

Reply:
fixed